test_communication/test/action_client_py.py

```diff
@@ -190,7 +192,10 @@ def is_result_valid(get_result_response):
     except BaseException:
         print('Exception in action client:', file=sys.stderr)
         raise
+    except ExternalShutdownException:
+        print('Action client stopped with sigterm', file=sys.stderr)
+        rc = 1
```

Comment:
@ivanpauno `BaseException` will always match `ExternalShutdownException`, `except` clauses need reordering.

Reply:
ah I didn't pay attention to that, thanks!
Fixed in https://github.com/ros2/system_tests/pull/490/commits/238321a89daf6a1351629c9bc9bd4bbff58f896f.